### DISCUSSION TITLE
automatic payment is not working

### DISCUSSION THREAD

**Discussion by richonguzman** [community/community#64025] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

I got payments from Sponsors on Stripe Express ready to be transferred for quite a while without knowing why has it last so long and can't find if there is a button to press or to process the payment

**Comment by bradshjg** [comment]
Howdy @richonguzman, thanks for reaching out! New Sponsors accounts are currently subject to a probation period of 60 days as documented in the [GitHub Sponsors terms](https://docs.github.com/en/site-policy/github-terms/github-sponsors-additional-terms#43-payment-timing) before we start monthly payout processing. Please feel free to reach out if you run into issues after the probationary period and excited to have you as part of Sponsors ❤️.

**Comment by richonguzman** [comment]
Hi

How much time is it to complete the 60 days?

Saludos,

Ricardo / Richon
(Enviado desde mi celular a tubos)

El 17-08-2023, a las 09:32, Jimmy Bradshaw ***@***.***> escribió:

﻿

Howdy @richonguzman<https://github.com/richonguzman>, thanks for reaching out! New Sponsors accounts are currently subject to a probation period of 60 days as documented in the GitHub Sponsors terms<https://docs.github.com/en/site-policy/github-terms/github-sponsors-additional-terms#43-payment-timing> before we start monthly payout processing. Please feel free to reach out if you run into issues after the probationary period and excited to have you as part of Sponsors ❤️.

**Reply by bradshjg** [reply]
The first payout should be initiated around Sept. 22nd (2023-09-22). Also love seeing folks doing cool things with packet radio ❤️.

**Comment by richonguzman** [comment]
Soon I will be doing an analog proyect for vhf aprs as well (with ax25 instead of Lora)

Saludos,

Ricardo / Richon
(Enviado desde mi celular a tubos)

El 17-08-2023, a las 10:01, Jimmy Bradshaw ***@***.***> escribió:

﻿

The first payout should be initiated around Sept. 22nd (2023-09-22). Also love seeing folks doing cool things with packet radio ❤️.
